internal/models/kubeconfig/grpc_gateway_runtime_error.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package models
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// GrpcGatewayRuntimeError grpc gateway runtime error
+//
+// swagger:model grpc.gateway.runtime.Error
+type GrpcGatewayRuntimeError struct {
```

Comment:
I don't see where this is being used

Reply:
And if this is removed, please remove GoogleProtobufAny too

Reply:
It's not used, It was generated. The first one can go away. I will need to think about GoogleProtobufAny.
I will merge this PR and address these unused structs in a new PR.